parm/fixed_files_mapping.yaml

```diff
@@ -139,35 +139,6 @@ fixed_files:
   !join_str ["FNSMCC | ",*FNSMCC],
   !join_str ["FNMSKH | ",*FNMSKH]
   ]
-  #"FNZORC | $FNZORC",
```

Comment:
This whole block was duplicated in this file, so I removed this one.

Reply:
This block is `FV3_NML_VARNAME_TO_FIXam_FILES_MAPPING`, while the one you removed is `FV3_NML_VARNAME_TO_SFC_CLIMO_FIELD_MAPPING`. It looks like the later is still being used in `ush/set_fv3nml_sfc_climo_filenames.py`, so I assume this will cause a failure. Am I missing something here?